installation/resources/installer-config-cluster.yaml.tpl

```diff
@@ -68,53 +64,14 @@ data:
 ---
 apiVersion: v1
 kind: ConfigMap
-metadata:
-  name: core-overrides
-  namespace: kyma-installer
-  labels:
-    installer: overrides
-    component: core
-data:
-  console.cluster.headerLogoUrl: "assets/logo.svg"
-  console.cluster.headerTitle: ""
-  console.cluster.faviconUrl: "favicon.ico"
-  minio.accessKey: "admin"
-  minio.secretKey: "topSecretKey"
-  minio.resources.limits.memory: 128Mi
-  minio.resources.limits.cpu: 250m
----
-apiVersion: v1
-kind: ConfigMap
 metadata:
   name: istio-overrides
   namespace: kyma-installer
   labels:
     installer: overrides
     component: istio
 data:
-  global.proxy.includeIPRanges: "10.0.0.1/8"
-
-  security.enabled: "true"
-
   gateways.istio-ingressgateway.loadBalancerIP: "__EXTERNAL_PUBLIC_IP__"
```

Comment:
We should also add (introduced in https://github.com/kyma-project/kyma/pull/2355):
```
 global.proxy.excludeIPRanges: "__PROXY_EXCLUDE_IP_RANGES__"
```